src/components/Reports/StaffExpenseReport/DownloadButtonGroup/downloadReport.test.tsx

```diff
@@ -0,0 +1,157 @@
+import { ReportType } from '../Helpers/StaffReportEnum';
+import { Transaction } from '../StaffExpenseReport';
+import { downloadCsv } from './downloadReport';
+
+const mockData: Transaction[] = [
+  {
+    category: 'transfer - withdrawal',
+    fundType: 'Primary',
+    month: '2025-07-01',
+    total: -2724,
+    __typename: 'BreakdownByMonth',
+  },
+  {
+    category: 'salary',
+    fundType: 'Secondary',
+    month: '2025-08-01',
+    total: 3500,
+    __typename: 'BreakdownByMonth',
+  },
+  {
+    category: 'expense reimbursement',
+    fundType: 'Primary',
+    month: '2025-09-01',
+    total: -500,
+    __typename: 'BreakdownByMonth',
+  },
+];
+
+const mockT = jest.fn((key: string) => key);
```

Comment:
Since we're not checking whether this function was called, it can just be a regular function. It doesn't need to be a `jest.fn()`.

```suggestion
const mockT = (key: string) => key;
```